admin-dev/themes/new-theme/.webpack/common.js

```diff
@@ -185,5 +186,14 @@ module.exports = {
       $: 'jquery', // needed for jquery-ui
       jQuery: 'jquery',
     }),
+    new CopyPlugin([
```

Comment:
```js
    new CopyPlugin([
      { from: 'scss/img' },
    ])
```

Reply:
In fact, no, because some images are ever copied with file-loader because they are used in scss files.

Reply:
oh, you're right :D